azure-cli-extensions.pyproj.user

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ProjectView>ProjectFiles</ProjectView>
+  </PropertyGroup>
+</Project>
```

Comment:
Why are there so many visual studio files added? are they all necessary?

Reply:
Intentionally, to support @yugangw-msft and my effort to make developing for extensions the same as the regular CLI. That means have a VS proj and sln file.

Reply:
there's 6 added visual studio related files?

Reply:
What can I say--it's visual studio 🤷‍♂️

Reply:
I cut back on the number of commit VS files. It should be 4 now.